### PR TITLE
Create 2020-04-06-coreweekly-14-2020

### PR DESCRIPTION
**Pull request by ttoine**
work in progress.
see https://github.com/PrestaShop/core-weekly-generator/pull/29 for the associated update of the projects list.

### CODE REVIEW

**Comment by matks** [comment]
@ttoine Dont hesitate to open issues on core-weekly-generator repository to fully automatize the process (so one day you can just click a button / run one CLI command this PR is created with only the "general message" section needed)

For example the "1.7.7.x (for v1.7.7.0)" could be automated

I cannot guarantee when we'll do it but having the issues list will be a good start

**Comment by ttoine** [comment]
Actually, it would be good to add the date and week number.

But, really, I still don't get it why it is interesting to add the branch detail like "(for v1.7.7.0)". People who follow the development and contribute knows that already. Unless you would like to add a link to a milestone or label on it? Or something else useful that I am missing?

**Comment by matks** [comment]
> Actually, it would be good to add the date and week number.
> 
> But, really, I still don't get it why it is interesting to add the branch detail like "(for v1.7.7.0)". People who follow the development and contribute knows that already. Unless you would like to add a link to a milestone or label on it? Or something else useful that I am missing?

A link to milestone could be really good indeed

I think the version numbers help because
- not all people are very familiar with the concept of branches (versus release), `1.7.6.x` "what is this x ?"
- it helps people to visualize what's coming
But the real reason I suggest adding them is 😅"people before us used to add it so let's continue doing this although we don't know why"

**Comment by matks** [comment]
@ttoine Your filename does not end with `.md` . I dont know if it's an issue for the blog, but github then does not render it correctly for file review

**Comment by ttoine** [comment]
thanks, you are right I missed the .md extension. And this would be an issue to render the blog, the build would certainly fail. It's corrected now.

I have also finished to edit the name of modules, and updated the PR on coreweekly generator too.

**Comment by ttoine** [comment]
this time the file name is updated with the .md at the end. re-thanks @matks

**Review by matks — APPROVED** [review]
Core Weekly is 👍 

Don't forget to update publication date before merging.

**Comment by ttoine** [comment]
> But the real reason I suggest adding them is sweat_smile"people before us used to add it so let's continue doing this although we don't know why"

So maybe it is not that important to continue to do it... Plus, sometimes, the milestone is not created already, so it would be impossible to add a link to it.

If you agree, I will change the publication date and merge the pr.

File: news/_posts/core-weekly/2020-04-06-coreweekly-14-2020.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 14 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-06-06
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-04-06-coreweekly-14-2020.md]:
```suggestion
date:   2020-06-07
```

**Pull request merged**